test/auro-datepicker.test.js

```diff
@@ -1,15 +1,10 @@
 import { fixture, html, expect } from '@open-wc/testing';
 import '../src/auro-datepicker.js';
+import '@aurodesignsystem/auro-calendar';
+import '@aurodesignsystem/auro-dropdown';
+import '@aurodesignsystem/auro-input';
 
 describe('auro-datepicker', () => {
-  it('sets the CSS class on auro-datepicker > div element', async () => {
-    const el = await fixture(html`
-      <auro-datepicker class="testClass"></auro-datepicker>
-    `);
-
-    expect(el.className).to.equal('testClass');
-  });
-
   // it('auro-datepicker is accessible', async () => {
```

Comment:
What's the plan for addressing this test?

Reply:
Tracked in a different ticket: https://github.com/AlaskaAirlines/auro-datepicker/issues/26